spec/predicates/deployment.md

```diff
@@ -0,0 +1,477 @@
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/deployment/v1",
+  "predicate": {
+
+    // Required: creation time.
+    "creationTime": "...",
+
+    // Optional: decision details.
+    "decisionDetails": {
+       "evidence": []<ResourceDescriptor>,
+       "policy": []<ResourceDescriptor>
+     },
+
+    // Optional: scopes.
+    "scopes": map[string]string{
+        "<scope-name1>/<version>": "value1",
+        "<scope-name2>/<version>": "value2"
+    }
+  }
+}
+
+```
+
+### Fields
+
+**`creationTime`, required** string ([Timestamp](https://github.com/in-toto/attestation/blob/main/spec/v1/field_types.md#Timestamp))
+
+The timestamp indicating what time the attestation was created.
+
+**`decisionDetails.evidence`, optional** (list of [ResourceDescriptor](https://github.com/in-toto/attestation/blob/main/spec/v1/resource_descriptor.md))
+
+List of evidence used to make a decision. Resources may include attestations or other relevant evidence.
+
+**`decisionDetails.policy`, optional** (list of [ResourceDescriptor](https://github.com/in-toto/attestation/blob/main/spec/v1/resource_descriptor.md))
+
+List of policies used to make the decision, if any.
+
+**`scopes`, optional** map of string to string (scope type to scope value)
+
+A set of protection scopes of different types. A protection scope identifies the deployment environment to be protected and binds the attestation subject (image, artifact) to it.
+A protection scope SHOULD identify the resources to be protected explicitly via their value (e.g., a service account, a Spiffe ID, a Kubernetes pod ID). A protection scope MAY identify the resource implicitly via an authorization / policy URI that hides these details.
+A scope has a type and a value. A type ends with its version encoded with `/version`, such as `/v1`. Examples of explicit scope types include Kubernetes's objects such as a pod's cluster ID or a GCP service account. Examples of implicit scope types incude [Google Cloud Binauthz](https://cloud.google.com/binary-authorization/) policy URIs.
+Let's see some examples:
+
+- If we want to "restrict an image to run only on GKE cluster X", the scope type is a "Kubernetes cluster" and "X" is the scope value.
+- If we want to "restrict an image to run only under service account Y", the scope type is "service account" and the scope value is "Y".
+- If we want to "run an image only if it has been vuln scanned", the scope is empty in the sense that the image is allowed to run in any target environment.
+
+### Parsing Rules
+
+This predicate follows the in-toto attestation [parsing rules](https://github.com/in-toto/attestation/blob/main/spec/v1/README.md#parsing-rules).
+Summary:
+
+- Consumers MUST ignore the `decisionDetails` field during verification. The field is purely informational and is intended only
+for troubleshooting and logging.
+- Consumers MUST reject attestations with scope types they do not recognize.
```

Comment:
That's correct. If the intention of the producer is to further restrict the environment where an artifact should be deployed, the verifier must understand all these "conditions", otherwise it would violate the producer's goal